<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Gary Twinn gary@twinn.net</author>
 <class>dialogNewBatch</class>
 <widget class="QDialog" name="dialogNewBatch">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>970</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>970</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New Batch</string>
  </property>
  <property name="windowIcon">
   <iconset resource="main.qrc">
    <normaloff>:/main/iconGTRun.svg</normaloff>:/main/iconGTRun.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt &quot;Segoe UI&quot;;</string>
  </property>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNew">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>create a new batch (will discard the current batch)</string>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioNewSimple">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>921</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Simple Batch (up to 8 steps). Allows user to define Line Blanks and Q-Standards as well as samples.</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioNewPlanchet">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>941</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>New Planchet. Allows the definition of each sample within a planchet, automatilally adds Line Blanks and Q-Standards. </string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEdit">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Edit the current batch</string>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMessage">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>911</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>btnEdit</tabstop>
  <tabstop>btnNew</tabstop>
  <tabstop>btnClose</tabstop>
  <tabstop>radioNewSimple</tabstop>
  <tabstop>radioNewPlanchet</tabstop>
 </tabstops>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
</ui>
